boards/msba2/include/periph_conf.h

```diff
@@ -79,8 +79,42 @@ extern "C" {
  * @name    UART configuration
  * @{
  */
+static const uart_conf_t uart_config[] = {
+    {
+        .dev = UART0,
+        .irq_prio_rx = 6,
+        .pinsel_rx   = 0,
+        .pinsel_tx   = 0,
+        .pinsel_msk_rx = BIT4,
+        .pinsel_msk_tx = BIT6,
+    },
+    {
+        .dev = UART1,
+        .irq_prio_rx = 6,
+        .pinsel_rx   = 4,
+        .pinsel_tx   = 4,
+        .pinsel_msk_rx = BIT3,
+        .pinsel_msk_tx = BIT1,
+    },
+    {
+        .dev = UART2,
+        .irq_prio_rx = 6,
+        .pinsel_rx   = 0,
+        .pinsel_tx   = 0,
+        .pinsel_msk_rx = BIT22,
+        .pinsel_msk_tx = BIT20,
+    },
+    {
+        .dev = UART3,
+        .irq_prio_rx = 6,
+        .pinsel_rx   = 9,
+        .pinsel_tx   = 9,
+        .pinsel_msk_rx = BIT26 | BIT27,
+        .pinsel_msk_tx = BIT28 | BIT29,
```

Comment:
I think this should be
```suggestion
        .pinsel_msk_tx = BIT24 | BIT25,
```

Reply:
Good catch!